bip-segwitaddress.mediawiki

```diff
@@ -0,0 +1,132 @@
+<pre>
+  BIP: x
+  Title: Address Formats for Witness Program
+  Author: Johnson Lau <jl2012@xbt.hk>
+  Status: Draft
+  Type: Standards Track
+  Created: 2015-12-24
+</pre>
+
+==Abstract==
+
+This BIP describes 2 new types of Bitcoin address to support native Segregated Witness (segwit) transactions. The first type resembles the original P2PKH base-58 address. The second type is a z-base-32 representation of a witness program with Damm algorithm checksum, which supports arbitrarily complex transactions, and is forward compatible to new version witness programs in the future.
+
+==Motivation==
+
+To define standard payment addresses for native segwit transactions to promote early adoption of the more efficient transaction method.
+
+== Specification ==
+=== P2PKH segwit address ===
```

Comment:
Why not also include a similar proposal for P2SH segwit while you're at it?

I assume that segwit being limited to 41 bytes means multisig etc. are planned to be P2SH.